storagenode/nodestats/cache.go

```diff
@@ -0,0 +1,161 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package nodestats
+
+import (
+	"context"
+	"math/rand"
+	"time"
+
+	"github.com/zeebo/errs"
+	"go.uber.org/zap"
+	"golang.org/x/sync/errgroup"
+
+	"storj.io/storj/internal/date"
+	"storj.io/storj/internal/sync2"
+	"storj.io/storj/pkg/storj"
+	"storj.io/storj/storagenode/reputation"
+	"storj.io/storj/storagenode/storageusage"
+	"storj.io/storj/storagenode/trust"
+)
+
+var (
+	// NodeStatsCacheErr defines node stats cache loop error
+	NodeStatsCacheErr = errs.Class("node stats cache error")
+)
+
+// Config defines nodestats cache configuration
+type Config struct {
+	MaxSleepDuration int           `help:"max number of seconds cache waits before performing each sync" releaseDefault:"300" devDefault:"1"`
```

Comment:
This should be a `time.Duration` then you don't have to mention the units

`"maximum duration to wait before requesting data"`